django_slack_oauth/views.py

```diff
@@ -1,43 +1,43 @@
 # -*- coding: utf-8 -*-
 
 import uuid
+from importlib import import_module
+
+from django.contrib.auth.mixins import LoginRequiredMixin
```

Comment:
Please format imports according to style guide

https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/